tests/test_deploy_scenarios.py

```diff
@@ -384,3 +384,51 @@ def test_deploy_with_update(
         logger.error(f"LogicException: {error.message}")
 
     deploy_fixture.check_log_stability()
+
+
+@pytest.mark.parametrize("on_schema_break", [OnSchemaBreak.AppendApp])
+def test_deploy_with_schema_breaking_change_append_in_mainnet(
+    deploy_fixture: DeployFixture, on_schema_break: OnSchemaBreak
+) -> None:
+    v1, _, v3 = get_specs()
+
+    app_v1 = deploy_fixture.deploy(v1, version="1.0", allow_delete=False, allow_update=False)
+    assert app_v1.app_id
+
+    try:
+        deploy_fixture.deploy(
+            v3,
+            version="2.0",
+            allow_delete=False,
+            allow_update=False,
+            on_schema_break=on_schema_break,
+        )
+    except DeploymentFailedError as error:
+        logger.error(f"DeploymentFailedError: {error}")
+    except LogicError as error:
+        logger.error(f"LogicException: {error.message}")
+
+    deploy_fixture.check_log_stability()
+
+
+@pytest.mark.parametrize("on_update", [OnUpdate.AppendApp])
```

Comment:
since there is only one value here, don't need to use parametrize, just use `OnUpdate.AppendApp` instead of the parameter value. Also remove `_in_mainnet` from the test name